proteinworkshop/config/encoder/egnn.yaml

```diff
@@ -2,7 +2,8 @@ _target_: "proteinworkshop.models.graph_encoders.egnn.EGNNModel"
 num_layers: 6 # Number of message passing layers
 emb_dim: 512 # Dimension of the node embeddings
 activation: relu # Activation function to use
-norm: layer # Normalisation layer to use
+norm: batch # Normalisation layer to use
 aggr: "sum" # Aggregation function to use
-pool: "sum" # Pooling operation to use
+pool: "mean" # Pooling operation to use
 residual: True # Whether to use residual connections
+drop_rate: 0.1 # Dropout rate
```

Comment:
Can we rename this to `dropout` for consistency with other models?

Reply:
Fixed